src/state/index.js

```diff
@@ -1,9 +1,17 @@
 import { createStore, applyMiddleware } from 'redux';
-import { composeWithDevTools } from 'redux-devtools-extension';
-import { reducer } from './reducers';
 import thunk from 'redux-thunk';
+import rootReducer from './reducers';
 
-// STORE
-const store = createStore(reducer, composeWithDevTools(applyMiddleware(thunk)));
+const middleWare = [thunk];
 
-export default store;
+export default createStore(rootReducer, applyMiddleware(...middleWare));
+
+// import { createStore, applyMiddleware } from 'redux';
+// import { composeWithDevTools } from 'redux-devtools-extension';
+// import { reducer } from './reducers';
```

Comment:
Commented code should be removed (especially if not used).

Reply:
Done!